src/block/block_addr.c

```diff
@@ -37,12 +49,11 @@ __block_buffer_to_addr(WT_BLOCK *block, const uint8_t **pp, uint32_t *objectidp,
      * that's simpler than testing sizes of 0 all over the place.
      */
     if (s == 0) {
+        *objectidp = 0;
         *offsetp = 0;
-        *objectidp = *sizep = *checksump = 0;
+        *sizep = *checksump = 0;
```

Comment:
These are all `uint32_t *` types. Why pull the initialization of just `objectidp` out?

Reply:
Habit, really -- setting them in the order they appear in the argument list, I find that way I don't worry about missing one.